notification/src/test/java/com/iluwatar/notification/CourseServiceTest.java

```diff
@@ -0,0 +1,20 @@
+package com.iluwatar.notification;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class CourseServiceTest {
+
+    @Test
+    void testCourseServiceTest() {
+        RegisterCourseDto registerDTO = new RegisterCourseDto();
+        registerDTO.setDepartment("English");
+        registerDTO.setCourseId("CS444");
+        registerDTO.setSemester("Fall21");
+        CourseService course = new CourseService();
+        Boolean isRegistered = course.registerCourse(registerDTO);
+        assertTrue(isRegistered);
+    }
```

Comment:
This probably tests that the happy scenario for registering for a course succeeds. The naming of the test could be better. Additionally, there should be more tests that check the negative cases (course registration does not succeed).